### PR TITLE
Add autofix to block-closing-brace-newline-after

### PR DESCRIPTION
**Pull request by ota-meshi**
> Which issue, if any, is this issue related to?

e.g. "Closes #000" or "None, as it's a documentation fix."

`block-closing-brace-newline-after` in #2829

> Is there anything in the PR that needs further explanation?

e.g. "No, it's self explanatory."

ex.

* option: `always***`

    code:
    ```css
    a { color: pink; } .b { color: red; }
    ```
    fixed:
    ```css
    a { color: pink; }
     .b { color: red; }
    ```

* option: `never***`

    code:
    ```css
    a { color: pink; }
    .b { color: red; }
    ```

    fixed:
    ```css
    a { color: pink; }.b { color: red; }
    ```

### CODE REVIEW

File: lib/rules/block-closing-brace-newline-after/index.js [hunk 1]
```diff
@@ -99,6 +99,22 @@ const rule = function(expectation, options) {
         index: -1,
         lineCheckStr: blockString(statement),
         err: msg => {
+          if (context.fix) {
+            if (expectation.indexOf("always") === 0) {
```
Comment (gucong3000) [inline lib/rules/block-closing-brace-newline-after/index.js]: Way not just use:
```js
if (expectation === "always") {
```

File: lib/rules/block-closing-brace-newline-after/index.js [hunk 2]
```diff
@@ -99,6 +99,22 @@ const rule = function(expectation, options) {
...
+              const index = nodeToCheck.raws.before.search(/\r?\n/);
+              if (index >= 0) {
+                nodeToCheck.raws.before = nodeToCheck.raws.before.slice(index);
+              } else {
+                nodeToCheck.raws.before =
+                  context.newline + nodeToCheck.raws.before;
+              }
+              return;
+            } else if (expectation.indexOf("never") === 0) {
```
Comment (gucong3000) [inline lib/rules/block-closing-brace-newline-after/index.js]: Way not just use:
```js
if (expectation === "never") {
```

File: lib/rules/block-closing-brace-newline-after/index.js (hunk 1 above)
Comment (hudochenkov) [reply lib/rules/block-closing-brace-newline-after/index.js]: @gucong3000 It matches `always`, `always-single-line`, `always-multi-line`.

File: lib/rules/block-closing-brace-newline-after/index.js (hunk 2 above)
Comment (hudochenkov) [reply lib/rules/block-closing-brace-newline-after/index.js]: @gucong3000 It matches `never-single-line`, `never-multi-line`.

**Comment by jeddy3** [comment]
> -   Added: `block-closing-brace-newline-after` autofix ([#3443](https://github.com/stylelint/stylelint/pull/3443)).

**Pull request merged**